### ISSUE TITLE
Headway might be more accurate than frequency

### ISSUE THREAD

**Issue by fjprobos** [Bondify/gtfs_functions#3]
Hi, maybe I am not understanding the objective and output of the function "lines_freq", but I think the fields identified as frequencies should be named as headways. 

![imagen](https://user-images.githubusercontent.com/3968323/108614195-73080380-73d7-11eb-8f92-1dc66506f7d0.png)

Both concepts are related but might induce errors if not accuratelly nammed. I suggest to keep both concepts in the computations of this function.

**Comment by Bondify** [comment]
hi @fjprobos , thanks for the feedback. Would you mind explaining a bit more what is the difference between both concepts and why it might induce to confusion?

I'm happy to make the changes but I'd like to understand the implications first. 

Cheers

**Comment by fjprobos** [comment]
Sure!

Frequency refers to the concept of "events in a period of time". In terms of transit refers to a number of expeditions observed at a certain point of the route during a period. For instance: 5 expeditions of line 601 in stop 15 during the rush hour. It is meassured in [events/time].

Headway refers to the concept of "time between succesive events". This means the amount of time between two succesive expeditions that pass along a certain point of the route. It is meassured in [time].

The average (armonic average) of a the headway can be computed as the inverse of the frequency:

headway=1/frequency

**Comment by d-wasserman** [comment]
As a practicing transportation planner I second this assessment. =)

**Comment by csdiehl** [comment]
I agree with this. The field identified as frequency is in fact headway

**Comment by Bondify** [comment]
hi everybody, after quite some research I ended up changing the name to `min_per_trip` which I understand is what `headway` represents. The reason why I chose `min_per_trip` instead of `headway` is because it's more clear for non-native english speakers. When I've worked with agencies around the globe, the term `headway` always raised some confusion. I hope this is ok with you. 

I'll close the issue for now.
